components/AppCookiePopup.vue

```diff
@@ -0,0 +1,31 @@
+<template>
```

Comment:
this component on mobile could take more space `w-full`

<img width="201" alt="Screenshot 2024-05-27 alle 17 08 47" src="https://github.com/logix-software/frontend-assessment/assets/12210564/a869bf2a-71b1-4143-a57e-1171fcdf586b">